src/test/java/seedu/address/logic/parser/SearchParserTest.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.logic.parser;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Set;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.IncorrectCommand;
+import seedu.address.logic.commands.SearchCommand;
+
+public class SearchParserTest {
+	private SearchParser parser;
+	@Before
+    public void setupParser() {
+        parser = new SearchParser();
+    }
+
+	@Test
+	/*public void parse(){
+		try{
+			assertParse("searchWord", "searchWord"); //Find how to pass a Set<String> in a method
```

Comment:
`Set` is an interface. You are probably looking for an implementation of that interface, such as `HashSet`?